source/core/Math.cpp

```diff
@@ -278,6 +276,16 @@ using namespace std;
 		return _NextInstruction();													\
 		}
 
+#define DECLARE_VIREO_COMPARISON_PRIMITIVES(TYPE) \
+    DECLARE_VIREO_PRIMITIVE2( IsLT0##TYPE, TYPE, Boolean, (_Param(1) = _Param(0) <  0) ) \
+    DECLARE_VIREO_PRIMITIVE2( IsGE0##TYPE, TYPE, Boolean, (_Param(1) = _Param(0) >= 0) ) \
```

Comment:
Why are we adding IsLT0 and IsGE0 but not IsLE0 and IsGT0? Do they also throw the same compiler warnings? If so, fixing partial compiler warnings for the same lines does not seem to be a good idea.